app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -153,8 +154,11 @@ class Scene extends React.Component {
 
   addBlocks = () => {
     const block = this.nextBlock();
-    const randomPositionBetweenWalls =
-      (this.state.width - wallWidth) * Math.random() + wallWidth;
+    const widthBetweenWalls =
+      this.state.width - wallWidth - parseInt(width, 10);
+    const randomDist =
+      (widthBetweenWalls * Math.random() + widthBetweenWalls * Math.random()) /
```

Comment:
This can be further reduced to: `widthBetweenWalls * (Math.random() + Math.random()) / 2`

I think it may also be beneficial to note in the commit message that this is an approximation of a normal distribution, not an actual normal distribution.